packages/replay/src/eventBuffer/WorkerHandler.ts

```diff
@@ -0,0 +1,106 @@
+import { logger } from '@sentry/utils';
```

Comment:
Note: I extracted this out of the `EventBufferCompressionWorker`, first because we may need it in follow ups (for error handling etc.), second I think it makes sense architecturally to have this separated - no need to leak this through the buffer implementation! Also allows us to easier encapsulate the id handling stuff etc.